db-setup/setup_database.bal

```diff
@@ -47,5 +49,36 @@ public function main() returns sql:Error? {
     _ = check mysqlClient->execute(`INSERT INTO social_media_database.posts(description, category, created_date, tags, user_id)
                                     VALUES ('I want to learn multi cloud', 'education', CURDATE(), 'gcp, aws, azure, infra, learn', 3);`);
 
+    // Add new column to `users` table in the database.
+    _ = check mysqlClient->execute(`ALTER TABLE social_media_database.users
+                                    ADD mobile_number varchar(15) DEFAULT NULL;`);
+
+    // Update records in `users` table with mobile number.
+    _ = check mysqlClient->execute(`UPDATE social_media_database.users
+                                    SET mobile_number = "+94771234123"
+                                    WHERE name = "Ranga";`);
+    _ = check mysqlClient->execute(`UPDATE social_media_database.users
+                                    SET mobile_number = "+94771234001"
+                                    WHERE name = "Ravi";`);
+    _ = check mysqlClient->execute(`UPDATE social_media_database.users
+                                    SET mobile_number = "+94771234002"
+                                    WHERE name = "Satish";`);                                    
+    _ = check mysqlClient->execute(`UPDATE social_media_database.users
```

Comment:
Instead we can directly change the insert operation right?

Reply:
Ack

Reply:
Fixed with [fb9aed53c86630c7fe312d54111274a4cf3a792c](https://github.com/shafreenAnfar/ballerina-social-media/pull/3/commits/fb9aed53c86630c7fe312d54111274a4cf3a792c)